src/routes.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import HomeScreen from 'features/onboarding/HomeScreen';
+import CreateJoinRoomScreen from 'features/onboarding/CreateJoinRoomScreen';
+
+export const APP_PATHS = {
```

Comment:
just a question for my own sake, so if someone explicitly types in crowdplay.ca/create, does it automatically render HomeScreen?